<ui version="4.0" >
 <class>FineReaderXMLDataParserWidget</class>
 <widget class="QWidget" name="FineReaderXMLDataParserWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FineReader XML Data Parser</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Parse XML File</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>XML file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="xmlFileNameLabel" >
        <property name="minimumSize" >
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>XML file ... </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="chooseXMLFileButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QCheckBox" name="showBBoxesCheckBox" >
        <property name="text" >
         <string>Show Bounding Boxes</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="saveBBoxesCheckBox" >
        <property name="text" >
         <string>Save Bounding Boxes as Images in:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="bboxSaveDirLabel" >
        <property name="text" >
         <string>select Directory...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QPushButton" name="chooseBBoxSaveDirButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3" >
       <widget class="QPushButton" name="parseXMLButton" >
        <property name="text" >
         <string>Parse!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="parseImageButton" >
     <property name="text" >
      <string>Parse Current Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="parseFolderButton" >
     <property name="text" >
      <string>Parse Folder</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
